cfpb/ccdb5-ui
# Improve complaint card link routing

The ellipses in complaint cards were using a vanilla anchor tag, causing them to ignore the app's router and lose the user's search status when returning to search results.

Addresses DATAP-1224

## Changes

- Replace anchor element with `Link` element.

## Testing

- `yarn test` should pass
- Search for something.
- Click the `[...]` on a complaint card with longer text.
- Click the `< Back to search results` link on the dedicated complaint page. It should take you back to your search results instead of the CCDB homepage.

## Screenshots

| before | after |
|--------|-------|
| ![before](https://github.com/cfpb/ccdb5-ui/assets/1060248/079a49ce-2e3e-48f9-bb7f-da100c440391) | ![after](https://github.com/cfpb/ccdb5-ui/assets/1060248/eb5cdacb-a4ba-4c59-9789-d4e2abbf2b9d) |

## Checklist
